### PR TITLE
update flush function type

### PR DESCRIPTION
**Pull request by rluvaton**

### CODE REVIEW

**Comment by mcollina** [comment]
@jsumners I did a git bisect and found this commit as a culprit: 
 [0d67bfa3d98d2d2076a3b1a9d9f6445a569c59a9](https://github.com/pinojs/pino-pretty/commit/0d67bfa3d98d2d2076a3b1a9d9f6445a569c59a9)

Does anything spot immediately?

**Comment by mcollina** [comment]
Taking a look now!

**Comment by mcollina** [comment]
https://github.com/pinojs/pino/pull/1825 fixes it

File: pino.d.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export interface LoggerExtras<Options = LoggerOptions> extends EventEmitter {
     /**
      * Flushes the content of the buffer when using pino.destination({ sync: false }).
      */
-    flush(): () => void;
+    flush(): void;
```
Comment (mcollina) [inline pino.d.ts:126]: can you add a test for this type change?

**Comment by jsumners** [comment]
> @jsumners I did a git bisect and found this commit as a culprit: [0d67bfa3d98d2d2076a3b1a9d9f6445a569c59a9](https://github.com/pinojs/pino-pretty/commit/0d67bfa3d98d2d2076a3b1a9d9f6445a569c59a9)
> 
> Does anything spot immediately?

Looks like you've figured it out. We shouldn't rely on internals of other modules like that. If we need them, we should export them from the other module in some way.

**Comment by jsumners** [comment]
Rebase and tests should pass.

**Comment by rluvaton** [comment]
Would appreciate adding the`HACKTOBERFEST-ACCEPTED` label once merged :)

File: pino.d.ts (hunk 1 above)
Comment (mcollina) [inline pino.d.ts:126]: Can you add a test for this type? We use tsd.

Comment (rluvaton) [reply pino.d.ts:126]: Oh, I did understand you meant that, np

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**